nucypher/network/nodes.py

```diff
@@ -1366,7 +1360,7 @@ def node_details(node):
 
         fleet_icon = node.fleet_state_nickname_metadata
         if fleet_icon is UNKNOWN_FLEET_STATE:
-            fleet_icon = "?"  # TODO
+            fleet_icon = "?"  # TODO  NRN, MN
```

Comment:
I assume `NRN == Not Right Now`, but `MN`? `Maybe Never`?

Reply:
https://getyarn.io/yarn-clip/27b10ac9-fbea-42bf-8944-21c0bd533de0